cli/args.go

```diff
@@ -115,6 +115,13 @@ func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, wri
 		return nil, err
 	}
 
+	envValue, envProvided := os.LookupEnv("TERRAGRUNT_IAM_ASSUME_ROLE_DURATION")
+	defaultIamAssumeRoleDuration := 3600 // Default STS sessions are 1 hour long (3600 seconds)
+	IamAssumeRoleDuration, err := parseIntArg(args, OPT_TERRAGRUNT_IAM_ASSUME_ROLE_DURATION, envValue, envProvided, defaultIamAssumeRoleDuration)
```

Comment:
NIT: remove `defaultIamAssumeRoleDuration` and use `options.DEFAULT_IAM_ASSUME_ROLE_DURATION` here instead.